test/docker-e2e/test/e2e/test_data/v3genesisAppState.json

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
what's the purpose of this file and where is it used?

Reply:
When starting a chain using Tastora for versions below v4, we need this file. Now since we test upgrades from app version 2 to 3 and 3 to 4 all using a v4 or v5 binary, this file is required. It was deleted some time ago. If we decide to omit those old app version compatibility test, we can delete this file. IMO it is nice to have all PRs tested with the old app versions.